WcaOnRails/app/assets/stylesheets/registrations.scss

```diff
@@ -25,3 +26,22 @@ table.registrations-table {
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
 }
+
+/* Styling bootstrap table sort */
+.fixed-table-loading {
+  display: none;
+}
+
+th {
+  &:focus { outline: none; }
+  .sortable {
+    &:hover { cursor: pointer; }
+    &:after {
+      font-family: FontAwesome;
+      content: '\f0dc';
+      margin-left: 5px;
+    }
+    &.asc:after { content: '\f0de'; color: #3f91eb; }
```

Comment:
Can you move this color to the new [variables.scss file](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/assets/stylesheets/variables.scss)?